package.json

```diff
@@ -21,7 +21,7 @@
     "@wordpress/hooks": "^2.12.0",
     "@wordpress/html-entities": "^2.7.0",
     "@wordpress/i18n": "^3.11.0",
-    "@wordpress/icons": "^2.10.3",
+    "@wordpress/icons": ">=5.0.0 <6.0.0",
```

Comment:
Why using `>=5.0.0 and <6.0.0`?

- Because a new icon `trendingUp` will be used in this PR, and it's only added [since 4.0.0][1].
- 6.0.0 removes the `minus` icon which is used in `@wp-g2/components` module version 0.0.140, so there are errors when using 6.0.0
- Therefore, 5.0.0 is the newest version that has no errors.

[1]: https://github.com/WordPress/gutenberg/blob/trunk/packages/icons/CHANGELOG.md#400-2021-05-20